pkg/controllers/nodetemplate/status_test.go

```diff
@@ -0,0 +1,669 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nodetemplate_test
+
+import (
+	"fmt"
+	"sort"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/samber/lo"
+	v1 "k8s.io/api/core/v1"
+	_ "knative.dev/pkg/system/testing"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+
+	"github.com/aws/karpenter-core/pkg/apis/v1alpha5"
+	. "github.com/aws/karpenter-core/pkg/test/expectations"
+
+	"github.com/aws/karpenter/pkg/apis/v1alpha1"
+)
+
+var _ = Describe("AWSNodeTemplate Status", func() {
```

Comment:
I think you can just keep these two suite_test files as one file in the same folder with different Describe() functions, I think it's actually easier to reason about since these two files are both pretty short compared to other suite_test files.